src/strat_engine/pool.rs

```diff
@@ -56,7 +56,7 @@ impl Pool for StratPool {
     }
 
     fn add_blockdev(&mut self, path: &Path) -> EngineResult<()> {
-        let bd = try!(BlockDev::new(&self.uuid, Path::new(path), MIN_MDA_SIZE, true));
+        let bd = try!(BlockDev::new(&self.uuid, path, MIN_MDA_SIZE, true));
```

Comment:
The near future plan is to change this method to ```add_blockdevs``` (plural). In that case, ```BlockDev::new``` would be the wrong method to use. The correct method to use would be ```batch_new```. In that case, there would be no place in the code that actually used ```BlockDev::new``` and therefore it would be nice if ```BlockDev::new``` would go away.